src/components/common/Switch.svelte

```diff
@@ -0,0 +1,239 @@
+<script>
+    export let label;
+    export let design = 'inner label'
+    export let options = [];
+	export let fontSize = 16;
+	export let value = 'on';
+
+    let checked = true;
+	const uniqueID = Math.floor(Math.random() * 100)
+
+    function handleClick(event){
+        const target = event.target
+        const state = target.getAttribute('aria-checked')
+        checked = state === 'true' ? false : true
+        value = checked === true ? 'on' : 'off'
```

Comment:
if the aria-checked value is 'true', the checked variable becomes false. It made me confused.  
and this line can be `checked ? 'on' : 'off' `

I do not have anything else about the code, thank you